src/frontend/src/common/utils/timeUtils.tsx

```diff
@@ -0,0 +1,9 @@
+export const formatTZDate = (d: string): string => {
```

Comment:
I think this util should have a line or two of documentation to say what it's doing. If you just know the func name, I don't think you'd guess what it actually does (I would assume it formats a date object as a string with TZ info, probably).

If I'm reading it correctly, I think this is an accurate docstring if you want to use it:
> Takes ISO8601 datetime with TZ and returns corresponding date (only) on user's machine.
> Eg, if user is in Pacific TZ, formatTZDate("2021-12-02T01:07:48+00:00) => "2021-12-01"

Also, might be worth it to tweak the name to something a bit more explicit. Trying not to be overly verbose, but maybe `datetimeWithTzToLocalDate`?